domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/offers/Settlement.java

```diff
@@ -153,10 +153,10 @@ private boolean lastStatementIsOffer() {
         return getLastStatement().getType().equals(StatementType.OFFER);
     }
 
-    private boolean lastStatementIsAcceptationNotBy(MadeBy madeBy) {
+    private boolean lastStatementIsNotAcceptationBy(MadeBy madeBy) {
         PartyStatement lastStatement = getLastStatement();
-        return lastStatement.getType().equals(StatementType.ACCEPTATION)
-            && !lastStatement.getMadeBy().equals(madeBy);
+        return !lastStatement.getType().equals(StatementType.ACCEPTATION)
+            || lastStatement.getMadeBy().equals(madeBy);
```

Comment:
I feel this was easier to understand before this change.

Reply:
Will revert and add a suppression